### ISSUE TITLE
[BUG] Incorrect values returned by app()

### ISSUE THREAD

**Issue by kluhan** [JoMingyu/google-play-scraper#102]
**google_play_scraper.VERSION**
'1.0.1'

**Describe the bug**
 Calling the app-function returns a dict with most expected fields containing None or random Lists

**Code**
```
>>> google_play_scraper.app('com.spotify.music')
{'title': None, 'description': None, 'descriptionHTML': None, 'summary': None, 'summaryHTML': None, 'installs': None, 'minInstalls': None, 'score': None, 'ratings': None, 'reviews': None, 'histogram': [0, 0, 0, 0, 0], 'price': None, 'free': None, 'currency': None, 'sale': False, 'saleTime': None, 'originalPrice': None, 'saleText': None, 'offersIAP': False, 'inAppProductPrice': None, 'size': None, 'androidVersion': None, 'androidVersionText': None, 'developer': None, 'developerId': None, 'developerEmail': None, 'developerWebsite': None, 'developerAddress': None, 'privacyPolicy': None, 'developerInternalID': None, 'genre': None, 'genreId': None, 'icon': None, 'headerImage': None, 'screenshots': [], 'video': None, 'videoImage': None, 'contentRating': None, 'contentRatingDescription': None, 'adSupported': None, 'containsAds': False, 'released': None, 'updated': None, 'version': [None, [[[[[None, [None, [[None, 2, [0, 0], [None, None, 'https://play-lh.googleusercontent.com/lvYCdrPNFU0Ar_lXln3JShoE-NaYF_V-DNlp4eLRZhUVkj00wAseSIm-60OoCKznpw'], None, None, None, None, None, 2], [None, 2, [0, 0], [None, None, 'https://play-lh.googleusercontent.com/lvYCdrPNFU0Ar_lXln3JShoE-NaYF_V-DNlp4eLRZhUVkj00wAseSIm-60OoCKznpw'], None, None, None, None, None, 2], [None, 2, [0, 0], [None, None, 'https://play-lh.googleusercontent.com/lvYCdrPNFU0Ar_lXln3JShoE-NaYF_V-DNlp4eLRZhUVkj00wAseSIm-60OoCKznpw'], None, None, None, None, None, 2]], 2, 2], 'SoundCloud - Play Music, Podcasts & New Songs', None, [[['SoundCloud', [None, None, None, None, [None, None, '/store/apps/developer?id=SoundCloud']], True]], [None, [None, [None, 'Stream music, save tracks, follow artists &amp; build playlists. All for free']]]], [], [[None, None, [None, ['4.7', 4.6666446]]]], [], [5, 4, 5], [None, None, None, None, [None, None, '/store/apps/details?id=com.soundcloud.android']], None, ['CAIaXAoeEhwKFmNvbS5zb3VuZGNsb3VkLmFuZHJvaWQQARgDEAAyEwiw+tX474TzAhVJ86QKHVGADPVyEwisxNb474TzAhUKtaQKHYp+BEWKAQ0IABIJCgVlbi1VUxAAqgJhGl8IABIeChwKFmNvbS5zb3VuZGNsb3VkLmFuZHJvaWQQARgDShMIsPrV+O+E8wIVSfOkCh1RgAz1mgETCKzE1vjvhPMCFQq1pAodin4ERfoBDwoNCAASCQoFZW4tVVMQAA=='], ['com.soundcloud.android', 7]], [None, [None, [[None, 2, [512, 512], [None, None, 'https://play-lh.googleusercontent.com/r55K1eQcji3QMHRKERq6zE1-csoh_MTOHiKyHTuTOblhFi_rIz06_8GN5-DHUGJOpn79'], None, None, None, None, None, 2], [None, 2, [512, 512], [None, None, 'https://play-lh.googleusercontent.com/r55K1eQcji3QMHRKERq6zE1-csoh_MTOHiKyHTuTOblhFi_rIz06_8GN5-DHUGJOpn79'], None, None, None, None, None, 2], [None, 2, [512, 512], [None, None, 'https://play-lh.googleusercontent.com/r55K1eQcji3QMHRKERq6zE1-csoh_MTOHiKyHTuTOblhFi_rIz06_8GN5-DHUGJOpn79'], None, None, None, None, None, 2]], 2, 2], 'Deezer Music Player: Songs, Playlists & Podcasts', None, [[['Deezer Music', [None, None, None, None, [None, None, '/store/apps/dev?id=8280508308326756579']], True]], [None, [None, [None, 'Deezer is a world of music in your pocket - listen anytime, anywhere! 🏃\u200d♂️🎶🌎']]]], [], [[None, None, [None, ['4.5', 4.5490284]]]], [], [5, 4, 5], [None, None, None, None, [None, None, '/store/apps/details?id=deezer.android.app']], None, ['CAIaWAoaEhgKEmRlZXplci5hbmRyb2lkLmFwcBABGAMQATITCLD61fjvhPMCFUnzpAodUYAM9XITCKzE1vjvhPMCFQq1pAodin4ERYoBDQgAEgkKBWVuLVVTEACqAl0aWwgBEhoKGAoSZGVlemVyLmFuZHJvaWQuYXBwEAEYA0oTCLD61fjvhPMCFUnzpAodUYAM9ZoBEwisxNb474TzAhUKtaQKHYp+BEX6AQ8KDQgAEgkKBWVuLVVTEAA='], ['deezer.android.app', 7]], [None, [None, [[None, 2, [512, 512], [None, None, 'https://play-lh.googleusercontent.com/mwTU9-4NX-_QlATb6lILSinKI47wAtOM38GjPYRPQSsQOG2hVRY4h9OfEhDxpf8ADQ'], None, None, None, None, None, 2], [None, 2, [512, 512], [None, None, 'https://play-lh.googleusercontent.com/mwTU9-4NX-_QlATb6lILSinKI47wAtOM38GjPYRPQSsQOG2hVRY4h9OfEhDxpf8ADQ'], None, None, None, None, None, 2], [None, 2, [512, 512], [None, None, 'https://play-lh.googleusercontent.com/mwTU9-4NX-_QlATb6lILSinKI47wAtOM38GjPYRPQSsQOG2hVRY4h9OfEhDxpf8ADQ'], None, None, None, None, None, 2]], 2, 2], 'Shazam: Discover songs & lyrics in seconds', None, [[['Apple, Inc.', [None, None, None, None, [None, None, '/store/apps/developer?id=Apple,+Inc.']], True]], [None, [None, [None, 'Find artists, lyrics, &amp; videos for free! Play or connect to stream music now!']]]], [], [[None, None, [None, ['4.8', 4.8311496]]]], [], [5, 4, 5], [None, None, None, None, [None, None, '/store/apps/details?id=com.shazam.android']], None, ['CAIaWAoaEhgKEmNvbS5zaGF6YW0uYW5kcm9pZBABGAMQAjITCLD61fjvhPMCFUnzpAodUYAM9XITCKzE1vjvhPMCFQq1pAodin4ERYoBDQgAEgkKBWVuLVVTEACqAl0aWwgCEhoKGAoSY29tLnNoYXphbS5hbmRyb2lkEAEYA0oTCLD61fjvhPMCFUnzpAodUYAM9ZoBEwisxNb474TzAhUKtaQKHYp+BEX6AQ8KDQgAEgkKBWVuLVVTEAA='], ['com.shazam.android', 7]], [None, [None, [[None, 2, [512, 512], [None, None, 'https://play-lh.googleusercontent.com/tLnLhJocCZI3wBtHGtBsASPK2omG-GxaozfsOCmgh5RRz95ux8YkGJhjpnKO5m3oCYQP'], None, None, None, None, None, 2], [None, 2, [512, 512], [None, None, 'https://play-lh.googleusercontent.com/tLnLhJocCZI3wBtHGtBsASPK2omG-GxaozfsOCmgh5RRz95ux8YkGJhjpnKO5m3oCYQP'], None, None, None, None, None, 2], [None, 2, [512, 512], [None, None, 'https://play-lh.googleusercontent.com/tLnLhJocCZI3wBtHGtBsASPK2omG-GxaozfsOCmgh5RRz95ux8YkGJhjpnKO5m3oCYQP'], None, None, None, None, None, 2]], 2, 2], 'Music Player - MP3 Player, Audio Player', None, [[['InShot Inc.', [None, None, None, None, [None, None, '/store/apps/developer?id=InShot+Inc.']], True]], [None, [None, [None, 'Best Free Music Player &amp; MP3 Player.\nPowerful Equalizer, 20 Themes, Tag Editor.']]]], [], [[None, None, [None, ['4.8', 4.836118]]]], [], [5, 4, 5], [None, None, None, None, [None, None, '/store/apps/details?id=musicplayer.musicapps.music.mp3player']], None, ['CAIaawotEisKJW11c2ljcGxheWVyLm11c2ljYXBwcy5tdXNpYy5tcDNwbGF5ZXIQARgDEAMyEwiw+tX474TzAhVJ86QKHVGADPVyEwisxNb474TzAhUKtaQKHYp+BEWKAQ0IABIJCgVlbi1VUxAAqgJwGm4IAxItCisKJW11c2ljcGxheWVyLm11c2ljYXBwcy5tdXNpYy5tcDNwbGF5ZXIQARgDShMIsPrV+O+E8wIVSfOkCh1RgAz1mgETCKzE1vjvhPMCFQq1pAodin4ERfoBDwoNCAASCQoFZW4tVVMQAA=='], ['musicplayer.musicapps.music.mp3player', 7]], [None, [None, [[None, 2, [512, 512], [None, None, 'https://play-lh.googleusercontent.com/WfnBxlXCNhKPaBBL6VasmJw2YTURoGDFxKlJmyGdz_0b49z2nJeiaN3iCXwbcEObvzg'], None, None, None, None, None, 2], [None, 2, [512, 512], [None, None, 'https://play-lh.googleusercontent.com/WfnBxlXCNhKPaBBL6VasmJw2YTURoGDFxKlJmyGdz_0b49z2nJeiaN3iCXwbcEObvzg'], None, None, None, None, None, 2], [None, 2, [512, 512], [None, None, 'https://play-lh.googleusercontent.com/WfnBxlXCNhKPaBBL6VasmJw2YTURoGDFxKlJmyGdz_0b49z2nJeiaN3iCXwbcEObvzg'], None, None, None, None, None, 2]], 2, 2], 'Music Player & MP3 Player - Lark Player', None, [[['Lark Player Studio', [None, None, None, None, [None, None, '/store/apps/developer?id=Lark+Player+Studio']], True]], [None, [None, [None, 'A music player that can play music offline, with themes, equalizer, lyrics etc.']]]], [], [[None, None, [None, ['4.7', 4.708595]]]], [], [5, 4, 5], [None, None, None, None, [None, None, '/store/apps/details?id=com.dywx.larkplayer']], None, ['CAIaWQobEhkKE2NvbS5keXd4LmxhcmtwbGF5ZXIQARgDEAQyEwiw+tX474TzAhVJ86QKHVGADPVyEwisxNb474TzAhUKtaQKHYp+BEWKAQ0IABIJCgVlbi1VUxAAqgJeGlwIBBIbChkKE2NvbS5keXd4LmxhcmtwbGF5ZXIQARgDShMIsPrV+O+E8wIVSfOkCh1RgAz1mgETCKzE1vjvhPMCFQq1pAodin4ERfoBDwoNCAASCQoFZW4tVVMQAA=='], ['com.dywx.larkplayer', 7]]], 'Similar', None, [None, None, None, None, [None, None, '/store/apps/collection/cluster?clp=ogofCBEqAggIMhcKEWNvbS5zcG90aWZ5Lm11c2ljEAEYAw%3D%3D:S:ANO1ljKzUPY&gsr=CiKiCh8IESoCCAgyFwoRY29tLnNwb3RpZnkubXVzaWMQARgD:S:ANO1ljJbTk4']], True, 2, None, [None, 'CjWC0_-4Ay8KJvqegZ0DIAgGEIPt-doNENrF_pIEEMOf2_cHEN24lK4BEKKb1vwGEMHtoIu_LxAFGiKiCh8IESoCCAgyFwoRY29tLnNwb3RpZnkubXVzaWMQARgD'], True], None, None, ['CBSqARUKEwiInNv474TzAhVJ86QKHVGADPU=']]], None, [], True], 'recentChanges': None, 'recentChangesHTML': None, 'comments': [**comments**], 'editorsChoice': False, 'appId': 'com.spotify.music', 'url': 'https://play.google.com/store/apps/details?id=com.spotify.music&hl=en&gl=us'}
```
Comments were manually removed and worked as expected.

**Expected behavior**
Returning a dict containing the values as in the AppStore

**Additional context**
As far as I have been able to investigate, it is due to a change in the GooglePlayStore. Can anyone confirm the behaviour I described?

**Comment by ucf0709** [comment]
Same here, i tried the script now and all values are none except comments

**Comment by JoMingyu** [comment]
It seems to be a problem caused by changing the data structure on the google play side. I will fix it quickly and upload a version.

**Comment by ucf0709** [comment]
Thanks for being interactive

**Comment by JoMingyu** [comment]
Issue fixed from version 1.0.2.
